plugins/parquet/README.md

```diff
@@ -78,4 +78,66 @@ To select the fields that you wish to partition your data on pass in a string of
 ParquetIO.HivePartition.Write(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
 
 ParquetIO.DirectoryPartition.Read(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
-```
\ No newline at end of file
+```
+# Apache Parquet/Arrow to ECL Type Mappings
+
```

Comment:
I see a couple issues with this table. Theoretically every Parquet type can be read in and automatically translated to any ECL type. The important translations are on output converting an ECL Record to an Arrow Schema i.e. the translations in fieldToNode.

I think this should be broken into two tables. The first table shows the Arrow types that are output for each datatype defined in a record and a second table showing the supported datatypes and it's "default" type (since they're not native they don't really have a default type, but some make more sense than others e.g fixed_size_binary vs Duration).